### ISSUE TITLE
[Adaptive Card iOS - ValueChangedAction.Json]: After expanding 'Country' combo box, on perfomring right swipe voice over focus is navigating on the background controls.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#180]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.4.0-beta.24.5.30.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS Version: 16.7.1
App Version: 1.0 (2.4.0-beta.24.5.30.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.5" control and select it.
5. Navigate to 'Scenarios' control and activate it.
6. Navigate to 'ValueChangedAction.json' and activate it.
7. Perform right swipe and navigate to the country combo box.
8. Double tap to activate it.
9. Perform right swipe and observe the issue.

**Actual Result:**
After expanding 'Country' combo box, on performing right swipe voice over focus is navigating on the background controls.

**Expected Result:**
After expanding the country combo box, voice over focus should be land on the menu items and on performing right swipe it should not lands on the background controls.

**Note:**
Same issue is repro throughout the application for the same type of control.

User Experience:
It will impact the screen reader user if the voice over does not convey the complete information for the day.

WCAG Reference:
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/94900d14-d8ce-435e-86dd-f63aa43d766f

### Card JSON

```JSON
NA
```
